started/index.md

```diff
@@ -125,7 +127,9 @@ The steps for installing with MSYS2 (recommended) are as follows:
 <div style="text-align: left" markdown="1">
 
 * You will need to install Go, gcc and the graphics library header files using the package manager.
-* `sudo apt-get install golang gcc libegl1-mesa-dev xorg-dev`
+* `sudo apt-get install golang gcc libegl1-mesa-dev xorg-dev libxkbcommon-dev`
+
+**NOTE:** Some Wayland compositors (Gnome's compositor Mutter etc.) do not support server side decorations resulting in Fyne applications showing up without window borders. Installing `libdecor` should solve these issues. 
```

Comment:
Raspberry Pi now uses Wayfire which seems to support server side rendering. So this should not be needed.

Reply:
Yes but you can also install something like Ubuntu 24.04 om Raspberry Pi and that ides Gnome. Maybe the RPi tab (which just checks arm64 right?) needs to be updated with more distribution options?